spacialgaze-plugins/misc.js

```diff
@@ -372,6 +372,13 @@ exports.commands = {
 			msg += '<button class="button" name="send" value="/emote add, ' + target[1] + ', ' + target[2] + '">Add emote</button></center>';
 			delete user.tokens[target[0]];
 			return SG.messageSeniorStaff(msg);
+		case 'disableintroscroll':
```

Comment:
This still dosent check to see if the room exists, or if the room already has its scroll bar disabled.